src/test/java/foodwhere/model/review/RatingTest.java

```diff
@@ -25,11 +25,15 @@ public void isValidRating() {
         assertThrows(NullPointerException.class, () -> Rating.isValidRating(null));
 
         // invalid contents
-        assertFalse(Rating.isValidRating(6)); // invalid number
-        assertFalse(Rating.isValidRating(7)); // invalid number
+        assertFalse(Rating.isValidRating("asdf")); // invalid number
+        assertFalse(Rating.isValidRating("7")); // invalid number
+        assertFalse(Rating.isValidRating("8")); // invalid number
+        assertFalse(Rating.isValidRating("")); // invalid number
+        assertFalse(Rating.isValidRating("abcd")); // invalid number
+        assertFalse(Rating.isValidRating("   ")); // invalid number
```

Comment:
I think I would check "03" (I would think valid?), "3." (probably invalid?), "3.0" (probably invalid?), "3 ", "3a", " 3", "a3", "-3", "-10000000000000000000", "10000000000000000000", "1180591620717411303427" (overflow cases and 2^70+3) too.

Reply:
Resolved in latest commit